gst/nnstreamer/tensor_crop/tensor_crop.c

```diff
@@ -51,13 +73,13 @@ GST_DEBUG_CATEGORY_STATIC (gst_tensor_crop_debug);
 enum
 {
   PROP_0,
-  PROP_SILENT
+  PROP_LATENESS
```

Comment:
Why silent is deleted?

Reply:
IMO, unnecessary. Actually this property is not used in tensor-crop.

Reply:
For debugging, we will need to printout a few information, controlled by slient.
E.g., printing cropping info

Reply:
updated

Reply:
Good! :)